GamerpotFrontend/src/library/constants/colors.ts

```diff
@@ -6,4 +6,5 @@ export const colors = {
   darkGray: '#3B3B3B',
   dealsBackground: '#161618',
   green: '#329C30',
+  tabBackground: '#161618',
```

Comment:
Why are you using the same color in two different variables? You can implement colors, based on which is the primary, the secondary, the tertiary (according to the importance of it)

Reply:
No, I did not notice that it already exists

Reply:
`tabBackground` has been removed and replaced by dealsBackground.